### PR TITLE
[zfs] docs: fix command to show zfs mount

### LINKED ISSUE #539: [ZFS] mount does not shown in talosctl
I updated to the latest talos version (v1.8.3) and kubernetes version (v1.31.3) and using the zfs extension. But unfortunatly the mount isn't shown with the command `talosctl get discoveredvolumes`.

```bash
talosctl get discoveredvolumes | grep

```

In the container the zfs pool is online and healthy

image
image

Can someone help me to solve this issue? I used the guide in the zfs extension page.

### PR DESCRIPTION
**Pull request by tzabbi**
- currently `talosctl get discoveredvolumes` is used but it's the wrong command
- correct is talosctl mounts

EDIT: closes #539

### CODE REVIEW

File: storage/zfs/README.md [hunk 1]
```diff
@@ -73,8 +73,6 @@ Next you have to create a pod which has privileged rights and prepare him for cr
 kubectl -n kube-system debug -it --profile sysadmin --image=alpine node/<node name>
 ```
 
-You are now in the container, install the required tools
```
Comment (smira) [inline storage/zfs/README.md:76]: I don't think this should be dropped

Comment (tzabbi) [reply storage/zfs/README.md:76]: Updated.

**Review by smira — APPROVED** [review]
thank you!

**Pull request merged**